documentation/_puppetdb_nav.html

```diff
@@ -39,6 +39,7 @@ <h2 id="puppetdb">PuppetDB {{ puppetdb_version }}</h2>
       <li><a href="/puppetdb/{{ puppetdb_version }}/scaling_recommendations.html">Scaling Recommendations</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/repl.html">Debugging with Remote REPL</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/load_testing_tool.html">Load Testing</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/resource_statuses.html">Resource Statuses **PE FEATURE**</a></li>
```

Comment:
So you've followed this schema elsewhere also, and I think the positioning is correct, I'm just worried that bold + caps will be like shouting. I was thinking literally: Resources Statuses (PE only) <- same concept a bit more subtle.